test/integration/Cardano/NetworkLayer/HttpBridgeSpec.hs

```diff
@@ -0,0 +1,102 @@
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Cardano.NetworkLayer.HttpBridgeSpec
+    ( spec
+    ) where
+
+import Prelude
+
+import Cardano.Launcher
+    ( Command (..), launch )
+import Cardano.NetworkLayer
+    ( NetworkLayer (..) )
+import Cardano.NetworkLayer.HttpBridge
+    ( HttpBridgeError (..) )
+import Cardano.Wallet.Primitive
+    ( Block (..), BlockHeader (..), Hash (..), SlotId (..) )
+import Control.Concurrent
+    ( threadDelay )
+import Control.Concurrent.Async
+    ( async, cancel )
+import Control.Exception.Base
+    ( ErrorCall )
+import Control.Monad.Except
```

Comment:
use rather `import Control.Monad.Trans.Except`

Reply:
@paweljakubas why is this so? Are we trying to use most explicit module in the import which exports the function?

Reply:
cf: https://github.com/input-output-hk/cardano-wallet/pull/81